src/js/node/tty.ts

```diff
@@ -139,17 +136,16 @@ Object.defineProperty(WriteStream, "prototype", {
       }
     };
 
-    var readline = undefined;
     WriteStream.prototype.clearLine = function (dir, cb) {
-      return (readline ??= require("node:readline")).clearLine(this, dir, cb);
+      return require("node:readline").clearLine(this, dir, cb);
     };
 
     WriteStream.prototype.clearScreenDown = function (cb) {
-      return (readline ??= require("node:readline")).clearScreenDown(this, cb);
+      return require("node:readline").clearScreenDown(this, cb);
     };
 
     WriteStream.prototype.cursorTo = function (x, y, cb) {
-      return (readline ??= require("node:readline")).cursorTo(this, x, y, cb);
+      return require("node:readline").cursorTo(this, x, y, cb);
     };
 
```

Comment:
The bits here aren't equiv. At least in the transform step I'm at. We currently don't hoist the `require("node:readline")` -> `@getInternalField(@internalModuleRegistry, 36) || @createInternalModuleById(36)` out.